examples/with-nextjs-and-clerk-auth/src/themes/monite.ts

```diff
@@ -0,0 +1,165 @@
+import type { Components } from '@mui/material/styles/components';
+import type {
+  Palette,
+  PaletteOptions,
+} from '@mui/material/styles/createPalette';
+import type { Theme, ThemeOptions } from '@mui/material/styles/createTheme';
+import type { TypographyOptions } from '@mui/material/styles/createTypography';
+import {
+  paletteDark as basePaletteDark,
+  paletteLight as basePaletteLight,
+  defaultMoniteComponents,
+  defaultMoniteTypography,
+} from '@team-monite/sdk-themes';
+
+const paletteLight: PaletteOptions = {
+  ...basePaletteLight,
+  primary: {
+    dark: '#1D59CC',
+    main: '#3737FF',
+    light: '#F4F8FF',
+  },
+  background: {
+    ...basePaletteLight.background,
+    menu: '#F1F2F5',
+    highlight: '#EBEBFF',
+  },
+  neutral: {
+    '10': '#111111',
+    '30': '#3B3B3B',
+    '50': '#707070',
+    '70': '#B8B8B8',
+    '80': '#DDDDDD',
+    '90': '#F3F3F3',
+    '95': '#FFFFFF',
+    '100': '#FFFFFF',
+  },
+};
+
+const paletteDark: PaletteOptions = {
+  ...basePaletteDark,
+  background: {
+    ...basePaletteDark.background,
+    menu: '#F1F2F505',
+  },
+  neutral: {
+    '100': '#111111',
+    '95': '#3B3B3B',
+    '90': '#707070',
+    '80': '#B8B8B8',
+    '70': '#DDDDDD',
+    '50': '#F3F3F3',
+    '30': '#FFFFFF',
+    '10': '#FFFFFF',
+  },
+};
+
+const typography:
+  | TypographyOptions
+  | ((palette: Palette) => TypographyOptions) = (palette: Palette) => ({
+  ...defaultMoniteTypography,
+  subtitle2: {
+    fontSize: '1.125rem',
+    fontWeight: 600,
+  },
+  body1: {
+    fontSize: '1rem',
+    fontWeight: 400,
+  },
+  body2: {
+    color: palette.neutral['10'],
+    fontSize: '0.875rem',
+    fontWeight: 500,
+  },
+  label1: {},
+  label2: {
+    fontSize: '0.875rem',
+    fontStyle: 'normal',
+    fontWeight: 500,
+  },
+  label3: {
+    color: palette.neutral['50'],
+    fontSize: '0.75rem',
+    fontStyle: 'normal',
+    fontWeight: 600,
+  },
+});
+
+const components: Components<Omit<Theme, 'components'>> = {
+  ...defaultMoniteComponents,
+  MuiDrawer: {
+    styleOverrides: {
+      root: {
+        '&.navigation-drawer .MuiPaper-root': {
+          borderRight: 0,
+        },
+      },
+    },
+  },
+  MuiList: {
+    styleOverrides: {
+      root: {
+        '&.navigation-list': {
+          margin: '0px 12px',
+
+          '.MuiListItem-root': {
+            marginTop: 8,
+          },
+
+          '.MuiListItemButton-root': {
+            borderRadius: 6,
+            padding: '8px 12px',
+          },
+
+          '.MuiListItemIcon-root': {
+            minWidth: 35,
+          },
+
+          '.MuiCollapse-root': {
+            marginTop: -8,
+            marginLeft: 12,
+          },
+        },
+      },
+    },
+  },
+  MuiButton: {
+    styleOverrides: {
+      root: {
+        '&.theme-select': {
+          borderRadius: 8,
+        },
+        '&.theme-select .theme-select__mode': {
```

Comment:
I think it's better to stick to the class naming style like the MUI, for example, we do the [same](https://github.com/team-monite/monite-sdk/blob/1c5be62c37a8916839a27956d35da336aa989765/packages/sdk-react/src/components/receivables/InvoiceStatusChip/InvoiceStatusChip.tsx#L88).
```
.ThemeSelect .ThemeSelect-modeLabel
```